### PR TITLE
flair: allow gz input

### PR DESCRIPTION
**Pull request by gregvonkuster**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/flair/test-data/fasta_indexes.loc [hunk 1]
```diff
@@ -1 +1 @@
-chrM	hg19	human mitochondrial genome	${__HERE__}/chrM.fa
+chrM	hg19	human mitochondrial genome	${__HERE__}/chrM.fa.gz
```
Comment (bernt-matthias) [inline tools/flair/test-data/fasta_indexes.loc:1]: Since this loc file is used by many tools I'm not sure if we should do this change.

Comment (bgruening) [reply tools/flair/test-data/fasta_indexes.loc:1]: ping @gregvonkuster

Comment (gregvonkuster) [reply tools/flair/test-data/fasta_indexes.loc]: Sorry for the delay on this - I was offline for a bit. :)

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Merci!

**Pull request merged**